huaweicloud/services/evs/resource_huaweicloud_evs_snapshot.go

```diff
@@ -50,6 +50,13 @@ func ResourceEvsSnapshotV2() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"metadata": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Computed: true,
```

Comment:
Please remove `Computed`,  the `Map` structure may trigger changes when backfilling.
Add a comment: // To avoid triggering changes metadata is not backfilled during read.